Gemfile

```diff
@@ -58,6 +58,7 @@ group :development do
   gem 'pry',                                :require => false
   gem 'json_pure', '<= 2.0.1',              :require => false if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.0.0')
   gem 'beaker-testmode_switcher'
+  gem 'rototiller', '~> 1.0.0',             :require => false
```

Comment:
Don't worry about this too much. If this takes off, we want to put it into `puppetlabs_spec_helper` (or its successor) and have it available automatically for everyone anyways.